server/sonar-server/src/main/java/org/sonar/server/es/StickyFacetBuilder.java

```diff
@@ -62,12 +63,21 @@ public StickyFacetBuilder(QueryBuilder query, Map<String, QueryBuilder> filters,
   }
 
   public AggregationBuilder buildStickyFacet(String fieldName, String facetName, Object... selected) {
-    return buildStickyFacet(fieldName, facetName, FACET_DEFAULT_SIZE, selected);
+    return buildStickyFacet(fieldName, facetName, t -> t, selected);
+  }
+
+  public AggregationBuilder buildStickyFacet(String fieldName, String facetName, Function<TermsBuilder, AggregationBuilder<?>> feature, Object... selected) {
```

Comment:
This method is complex. It deserves a javadoc (I know, other methods are not documented for legacy reasons...)